src/docs/asciidoc/cdc-configuration.adoc

```diff
@@ -595,6 +595,87 @@ The Eventuate Tram CDC supports `eventuate-tram` and `eventuate-local`
 
 |===
 
+==== Configuring a cleaner
+
+Cdc can automatically remove old messages from message table and received_message table.
```

Comment:
This should explain that you configure one or more named cleaners.
Each clean can clean a message table and/or a received_message table.

Reply:
ok, I will add that